src/components/IndexTable/IndexTable.tsx

```diff
@@ -38,7 +39,7 @@ export interface IndexTableHeading {
 }
 
 export interface IndexTableBaseProps {
-  headings: IndexTableHeading[];
+  headings: NonEmptyArray<IndexTableHeading>;
```

Comment:
Since we access headings via index then dot notation we'll want to type the array to have headings